[DOCS] Update GX Cloud Documentation to Reflect New Data Asset Workflow (#9694)
great-expectations/great_expectations: docs/docusaurus/docs/cloud/connect/connect_postgresql.md

- You have stopped all local running instances of the GX Agent.

## Connect to a PostgreSQL Data Asset

1. In GX Cloud, click **Data Assets** > **New Data Asset**.

2. Copy the code in the code pane.

3. Prepare your PostgreSQL environment:

   - In pgAdmin, select a database.

   - Click **Tools** > **Query Tool**.

   - Paste the code you copied in step 2 into the **Query** pane to create and assign the `gx_role` role and allow GX Cloud to access to all `public` schemas and tables on a specific database.

      Replace `<your_password>` and `<your_database>` with your own values. `ALTER DEFAULT PRIVILEGES IN SCHEMA public GRANT SELECT ON TABLES TO gx_role;` is optional and gives the `gx_role` user access to all future tables in the defined schema.

    - Click **Execute/Refresh**.

4.  In GX Cloud, click **I have created a GX Cloud user with valid permissions** and then click **Continue**.

> 5. Enter a meaningful name for the Data Asset in the **Data Source name** field.

Reviewer: This is technically the name for the Data Source. The data asset names are defined later

Author: @annabarr I have updated the text.

6. Enter a connection string in the **Connection string** field. The connection string format is `postgresql+psycopg2//YourUserName:YourPassword@YourHostname:5432/YourDatabaseName`. 

7. Optional. Select **Test connection** to test the Data Source connection. Testing the connection to the Data Source is a preventative measure that makes sure the connection configuration is correct. This verification can help you avoid errors and can reduce troubleshooting downtime.

8. Click **Continue**.

9. Select **Table Asset** or **Query Asset** and complete the following fields:

>     - **Table name**: When **Table Asset** is selected, enter a name for the table you're creating in the Data Asset.

Reviewer 2: This seems a little unclear.  You aren't so much entering a name for the table you're creating as you're entering the name of the table the Data Asset connects to.

Author: @Rachel-Reverie, I attempted to clarify the text.

>     - **Table name**: When **Table Asset** is selected, enter a name for the table you're creating in the Data Asset.

Reviewer 2: Maybe:
```suggestion
    - **Table name**: When **Table Asset** is selected, enter the name of the table you're connecting to with the Data Asset.
```

    
    - **Data Asset name**: Enter a name for the Data Asset. Data Asset names must be unique across all Data Sources in GX Cloud.

>     - **Query**: When **Query Asset** is selected, enter the query that you want to run on the table. 

Reviewer 2: I think this is slightly inaccurate.  When query asset is selected, there is no "the table" specified by the user.  Instead, since the query could theoretically join multiple tables, it should be something like:
```suggestion
    - **Query**: When **Query Asset** is selected, enter the query that you want to run on the database.
```

Author: @Rachel-Reverie, I attempted to clarify the text.
